testsuite/MDAnalysisTests/test_failure.py

```diff
@@ -0,0 +1,26 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning,
+# Oliver Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+import os
+
+
+def test_failure():
+    """Fail if the MDA_FAILURE_TEST environment variable is set.
```

Comment:
On 13/06/16 12:19, kain88-de wrote:
>
> In testsuite/MDAnalysisTests/test_failure.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/874#discussion_r66769392>:
>
> > +# Oliver Beckstein and contributors (see AUTHORS for the full list)
> > +#
> > +# Released under the GNU Public Licence, v2 or any higher version
> > +#
> > +# Please cite your use of MDAnalysis in published work:
> > +#
> > +# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
> > +# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
> > +# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
> > +#
> > +
> > +import os
> > +
> > +
> > +def test_failure():
> > +    """Fail if the MDA_FAILURE_TEST environment variable is set.
>
> what is this variable and why is this test included now?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/874/files/50bd3ea09e9ceaaae4b2eea4856c20612ac85d14#r66769392>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ABUWus0BwcwjKBTPxQNUN2OgdOnUgRlGks5qLS6wgaJpZM4IzFwH>.
>
This test allows to have at least one failing test on command. Just set 
the MDA_FAILURE_TEST environment variable and this test will fail. 
Thanks to that very useful tool, you do not have to sabotage a test to 
see if your plugin works on test failures.